wifiphisher/common/interfaces.py

```diff
@@ -623,13 +623,66 @@ def on_exit(self):
                 mac_address = adapter.original_mac_address
                 self.set_interface_mac(interface, mac_address)
 
+        if (hasattr(enable_network_manager, 'has_disable_nm') and
+                enable_network_manager.has_disable_nm):
+            # recover the network manager if we have disabled it past
+            enable_network_manager(True)
+
+
+def enable_network_manager(nm_enable):
```

Comment:
Perhaps `power_network_manager` would be a better name since enable implies that you only have the option to enable it and not disable.